code/baseStation/base_parser3.py

```diff
@@ -15,55 +23,59 @@
 
 THERMAL_HEIGHT = 32
 
+MSGPACK_BUFFER_LINE_LENGTH = 30
```

Comment:
A little explanation:
First of all, we agreed to packets of length 25 due to the fact, that thermal image line length is 24 bytes (25 with header)
Why 30?
Mergeblock does not control its output length, so it could write blobs of 24+ bytes. If these would exceed the blob length, LoRa block could get confused for a while. Of course everything is prepared for such scenarios via try/except, however, if we make the blob slightly longer that's going to happen rarely.
Lora block strips these blobs to finally send 25-byte packets.